stl/src/parallel_algorithms.cpp

```diff
@@ -25,7 +25,13 @@ extern "C" {
 
 // TRANSITION, ABI

```

Comment:
We can no longer remove this function in vNext without breaking the library, so this `TRANSITION` comment is no longer appropriate.

Reply:
```suggestion
```